### PR TITLE
rewrite the tasks using YAML syntax

### PR DESCRIPTION
**Pull request by arbabnazar**
@edx/devops kindly review. Thanks

### CODE REVIEW

File: playbooks/roles/snort/tasks/main.yml [hunk 1]
```diff
@@ -1,47 +1,50 @@
...
-  shell: oinkmaster -C /etc/oinkmaster.conf -o /etc/snort/rules/ 
+- name: Configure snort and snort (debian)
+  template:
+    src: "etc/snort/{{ item }}"
+    dest: "/etc/snort/{{ item.rstrip('.j2') }}"
+    owner: root
+    group: root
+    mode: "0644"
+  with_items:
+    - snort.conf.j2
```
Comment (e0d) [inline playbooks/roles/snort/tasks/main.yml]: @arbabnazar  I'd like to standardize on using anonymous dicts with user,group,mode, etc.  If you have time.

File: playbooks/roles/snort/tasks/main.yml [hunk 2]
```diff
@@ -1,47 +1,50 @@
...
+    name: snort
+    state: started
+
+- name: Open read permissions on snort logs
+  file:
+    name: "/var/log/snort"
+    state: directory
+    mode: "0755"
+
+- name: Install oinkmaster cronjob
```
Comment (e0d) [inline playbooks/roles/snort/tasks/main.yml]: If above is refactored, this can move up.

**Comment by e0d** [comment]
Small comments, but generally :+1:

**Comment by arbabnazar** [comment]
@e0d please review. Thanks

**Comment by arbabnazar** [comment]
@e0d please review. Thanks

**Comment by arbabnazar** [comment]
@e0d please review and merge this one if it looks OK after addressing your comments. Thanks

**Pull request closed without merging**